lib/landrush/action/setup.rb

```diff
@@ -15,24 +15,35 @@ def call(env)
         # This is after the middleware stack returns, which, since we're right
         # before the Network action, should mean that all interfaces are good
         # to go.
-        record_machine_dns_entry if enabled?
-        setup_static_dns if enabled?
+        post_boot_setup if enabled?
       end
 
       def pre_boot_setup
         record_dependent_vm
         add_prerequisite_network_interface
-        setup_host_resolver
         configure_server
         start_server
       end
 
+      def post_boot_setup
+        record_machine_dns_entry
+        setup_static_dns
+        setup_host_resolver(env)
+      end
+
       def record_dependent_vm
         DependentVMs.add(machine_hostname)
       end
 
-      def setup_host_resolver
-        ResolverConfig.new(env).ensure_config_exists!
+      def setup_host_resolver(env)
```

Comment:
There is need, but no code for it yet. Linux will continue to require manual configuration. Linux is also harder in a sense that we most likely have to differentiate between host types :-( OS X and Windows are easier in this regards. But adding support for Linux is definitely on the roadmap. Want to help!? ;-)